packages/ui-app/src/styles/semantic.css

```diff
@@ -190,6 +190,10 @@
   }
 }
 
+.ui--query {
```

Comment:
Please move this specifically to the explorer (This area is reserved for specific semantic-ui overrides). Additionally, would prefer to stick to `rem` where possible.

Reply:
OK. I don't have so much experience with CSS namespacing and overriding, so please don't hesitate to comment on the result if it's not exactly what you expect. That way I get to learn.

Reply:
No issues, just move it to `app-explorer/index.css` (And adjust the naming as well to just align with what is in there)